itests/standalone/smoke/src/test/java/org/wildfly/camel/test/smoke/ContextTrackerRegistryTest.java

```diff
@@ -51,8 +52,8 @@
 public class ContextTrackerRegistryTest {
 
     @Deployment
-    public static JavaArchive createDeployment() {
-        return ShrinkWrap.create(JavaArchive.class, "camel-context-tracker-tests.jar")
+    public static WebArchive createDeployment() {
+        return ShrinkWrap.create(WebArchive.class, "ContextTrackerRegistryTest.war")
```

Comment:
Right, this is for the same reason as we did with CXF. For CXF we said that's OK, because who who wants to expose web endpoints should bother to use a web archive.